Jellyfin.Server/Program.cs

```diff
@@ -657,5 +652,48 @@ private static string NormalizeCommandLineArgument(string arg)
 
             return "\"" + arg + "\"";
         }
+
+        private static string GetUnixSocketPath(IConfiguration startupConfig, IApplicationPaths appPaths)
+        {
+            var socketPath = startupConfig.GetUnixSocketPath();
+
+            if (string.IsNullOrEmpty(socketPath))
+            {
+                var xdgRuntimeDir = Environment.GetEnvironmentVariable("XDG_RUNTIME_DIR");
+                if (xdgRuntimeDir == null)
+                {
+                    // Fall back to config dir
+                    socketPath = Path.Join(appPaths.ConfigurationDirectoryPath, "socket.sock");
+                }
+                else
+                {
+                    socketPath = Path.Join(xdgRuntimeDir, "jellyfin-socket");
```

Comment:
Can we have the socket name be `jellyfin.socket` in both cases?

Reply:
if we're willing to break the previous behavior, I'm for changing it to `jellyfin.sock`. It's similar to `mysqld.sock`, `php7.4-fpm.sock` etc.